requirements.txt

```diff
@@ -2,5 +2,6 @@ mkdocs==1.2.3
 mkdocs-material==8.1.3
 pymdown-extensions==9.1
 markdown==3.3.6
+mike==1.1.2
```

Comment:
Mike is back! ;)

Reply:
![HahaMickaelJacksonGIF](https://user-images.githubusercontent.com/1822123/148545098-69125bd1-618b-4f17-883e-96e03660e9c4.gif)